common/src/test/java/org/opensearch/ml/common/model/MLDeployingSettingTests.java

```diff
@@ -109,12 +109,12 @@ public void parseWithIllegalArgumentInteger() throws Exception {
 
     @Test
     public void parseWithIllegalField() throws Exception {
-        final String expectedInputStrWithIllegalField = "{\"is_auto_deploy_enabled\":true," +
+        final String expectedInputStrWithIllegalField = "{\"is_auto_deploy_enabled\":true," + "\"model_ttl_hours\":0," +
                 "\"illegal_field\":\"This field need to be skipped.\"}";
 
         testParseFromJsonString(expectedInputStrWithIllegalField, parsedInput -> {
             try {
-                assertEquals(expectedInputStr, serializationWithToXContent(parsedInput));
+                assertEquals("{\"is_auto_deploy_enabled\":true,\"model_ttl_minutes\":-1}", serializationWithToXContent(parsedInput));
```

Comment:
What if `model_ttl_minutes` is 0 ?

Reply:
Just the opposite, with model_ttl_minutes == 0, the model will be removed in the next cron job running cycle right away. By default, without any given value for this ttl in minutes, model_ttl_minutes == -1 and it will make the model never expire.